src/app/images-collection/images-collection-detail/images-collection-detail.component.html

```diff
@@ -38,8 +38,18 @@ <h3 class="wipp-title">Collection detail</h3>
               {{sourceJob.name}}
             </a>
           </ng-template>
-          <ng-template #uploadedTemplate>
-            Uploaded
+          <ng-template #importTemplate>
+            <div *ngIf="imagesCollection.importMethod == 'CATALOG'">
+              <a [href]="sourceCatlaogLink"
+                target="_blank"
+                placement="left"
+                class="wipp-link">
+                {{imagesCollection.sourceCatalog}}
+              </a>
+            </div>
+            <div *ngIf="imagesCollection.importMethod == 'UPLOADED' || imagesCollection.importMethod  == null">
+              Uploaded
+            </div>
```

Comment:
- Typo in `sourceCatlaogLink`
- In the link for Catalog: 
  - class `wipp-link` is not needed here (this is a regular link), 
  - there is a tooltip placement but no tooltip -> add a tooltip "Go to Catalog" with external link icon (see example is csv detail collection for plot link)
  - the text displayed for the link should not be `{{imagesCollection.sourceCatalog}}` (which would show a partial url not really readable by the user) but something generic, such as "Catalog"
- for the Uploaded part, it is best to check for null first (`!imagesCollection.importMethod`) and then if importMethod is Uploaded